extension/src/main/scala/io/scalac/mesmer/extension/metric/DispatcherStaticMetricsMonitor.scala

```diff
@@ -0,0 +1,13 @@
+package io.scalac.mesmer.extension.metric
+
+import io.scalac.mesmer.core.AttributesSerializable
+import io.scalac.mesmer.core.model._
+import io.scalac.mesmer.core.module.AkkaDispatcherModule
+
+object DispatcherStaticMetricsMonitor {
+  final case class Attributes(node: Option[Node], executor: Executor, minThreads: MinThreads, maxThreads: MaxThreads, parallelismFactor: Parallelism) extends AttributesSerializable {
+    val serialize: RawAttributes = node.serialize ++ executor.serialize ++ minThreads.serialize ++ maxThreads.serialize ++ parallelismFactor.serialize
+  }
+
+  trait BoundMonitor extends Bound with AkkaDispatcherModule.Metrics[MetricObserver[Long, Attributes]] {}
+}
```

Comment:
which metric type is best to use for static config metric? MetricObserver? Counter? Histogram?

Reply:
Some follow-up questions: 
 
 - for min/max values we were using eg. Gauge + some internal pre-aggregation afaik. See [processingTimeMin](https://github.com/ScalaConsultants/mesmer-akka-agent/blob/0b9207fb27932b483c6dba694679a66a1c1a29c6/extension/src/main/scala/io/scalac/mesmer/extension/upstream/OpenTelemetryActorMetricsMonitor.scala#L220) or [processingTimeMax](https://github.com/ScalaConsultants/mesmer-akka-agent/blob/0b9207fb27932b483c6dba694679a66a1c1a29c6/extension/src/main/scala/io/scalac/mesmer/extension/upstream/OpenTelemetryActorMetricsMonitor.scala#L227). Not 100% sure if this should be done the same way though. Could you check if opentelemetry Histogram registers min/max? If yes, then I'd probably use histogram for these values.  
 - what kind of instruments does e.g. Kamon use for this case? Does it support these particular metrics?